NOTES.ANDROID

```diff
@@ -46,8 +46,8 @@
  One can engage clang by adjusting PATH to cover NDK's clang. Just keep
  in mind that if you miss it, Configure will try to use gcc... Also,
  PATH would need even further adjustment to cover unprefixed, yet
- target-specific, ar and ranlib (or not, if you use binutils-multiarch
- on your Linux).
+ target-specific, ar and ranlib. (Or not, if you use binutils-multiarch
+ on your Linux.)
```

Comment:
I disagree with making the parenthesis a separate sentence.

Reply:
Trouble is that it screams for punctuation mark after ranlib, and period is the only possibility. Of course provided that we want to keep parenthesis. And we do, because it's kind of a hint of possibility, rather than something we want to actually stand behind and recommend.

Reply:
I'm not sure how it "screams for punctuation mark" (and that it has to be immediate). I do know that parentheses as their own sentence are bad form in many written natural languages (such as Swedish, French and English, which are the ones I know well enough)

Reply:
> how it "screams for punctuation mark"

It looks like ranlib-specific remark. As if you don't have to care specifically about ranlib if you have binutils-multiarch.

> parentheses as their own sentence are bad form

Sure. But it's not high-quality literature we are talking about here. Here intention with parenthesis is to say "don't quote me on that" or "provided that you understand what you are doing". But those are too personal/official for a casual remark like one in question. BTW, you know which parenthesis I would prefer, right? :-) :-) :-)

Reply:
Maybe omit parenthesis and put it as "Or **perhaps** not, ..."?

Reply:
"It's possible that you don't need to bother, if..."

Reply:
For English, at least, "bad form" is entirely in the eye of the beholder, and what is "correct usage" is to large extent "whatever you can get away with".
(I say, being one who uses full-sentence parentheticals with some regularity.)

Reply:
Ok...  you learn something new every day...  But then, we found another solution, so ;-)